argo-cd-apps/base/enterprise-contract.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: rekor
+
+spec:
+  project: default
+  source:
+    path: helm-charts/rekor
+    repoURL: 'https://github.com/sigstore/sigstore-helm-operator'
```

Comment:
Should this be 
`path: components/enterprise-contract-service`
`repoURL: https://github.com/redhat-appstudio/infra-deployments.git`

Then have the rekor configs under `path`?